content/StaticMethodsAndProperties/StaticMethodsAndProperties.facts.md

```diff
@@ -0,0 +1,68 @@
+### Facts for Static Methods and Properties lesson:
+
+Static method and property are special types of methods and properties that can be called without creating an object as an instance of a class.
+
+The `static` keyword is used to define a static method and property.
+
+To declare a static method, add the `static` keyword after the visibility keyword followed by the `function` keyword; the method name; a pair of parentheses `()` which may contain the parameters; and a method body enclosed in curly braces `{}`.
+
+```php
+public static function myStaticMethod() { }
+```
+
+To declare a static property, add the `static` keyword after the visibility keyword like `public` followed by the property name.
+
+```php
+public static myStaticProperty;
+```
+
+A class name with the scope resolution operator `::` is used to access static methods and properties with `public` visibility outside a class.
+
+```php
+MyClass::myStaticProperty;
+MyClass::myStaticMethod();
+```
+
+The `self` keyword with the scope resolution operator `::` is used to access static methods and properties within a class itself.
+
+The `$this` pseudo-variable is not allowed to use inside a static method.
+
+The object operator `->` cannot be used to access static properties.
+
+Code:
+
+```php
+<?php
+class MyClass 
+{
+    public static $myStaticProperty;
+
+    public static function myStaticMethod()
+    {
+        echo "This is a static method." . self::$myStaticProperty;
+    }
+}
+
+MyClass::$myStaticProperty = "This is a static property.";
+MyClass::myStaticMethod();
+?>
+```
+
+Output:
+
+```
+This is a static method.This is a static property.
```

Comment:
This is possible but I opt not to add a space since this outputs comes from different sources (the method and property value) I want to show the learners only the result of accessing the method and property of the class without altering the returned result. Thank you.